src/lib/components/checkbox/stories/index.stories.tsx

```diff
@@ -19,14 +14,35 @@ type Story = StoryObj<typeof Checkbox>;
 
 export const Normal: Story = {
```

Comment:
Normal checkbox isn't working as expected, when I click on normal checkbox to enable or disabled it's work on default checkbox